lib/bookmark_query.rb

```diff
@@ -71,6 +71,8 @@ def user_bookmarks
     Bookmark.where(user: @user)
       .includes(post: :user)
       .includes(post: { topic: :tags })
+      .includes(topic: :topic_users)
       .references(:post)
+      .where(topic_users: { user_id: [@user.id, nil] })
```

Comment:
Why do we need `nil` here? `TopicUser#user_id` is a not nullable column so the `nil` filter will not have any effect.